tutors3/plugin.py

```diff
@@ -9,8 +9,8 @@
 config = {
     "defaults": {
         "VERSION": __version__,
-        "HOST": "s3.amazonaws.com",
-        "PORT": "443",
+        "HOST": "",
```

Comment:
I guess you could just set these defaults to `None`, and then instead of:

```
{% if S3_HOST != "" and S3_PORT != "" %}
```

you could use 

```
{% if S3_HOST and S3_PORT %}
```

which is a little more compact.